packages/extension-chakra-storefront/src/theme/components/project/footer.js

```diff
@@ -52,11 +57,14 @@ export default {
             fontSize: 'sm',
             marginBottom: 4
         },
-        localeSelector: {
-            display: 'inline-block',
+        localeSelectorWrapper: {
+            width: 'auto',
+            maxWidth: '246px',
```

Comment:
What is this arbitrary width?

Reply:
Without maxWidth, the select will expend full width. Using "inline-block" won't work well with Chakra v3 so I just input current maxWidth value to cap the width. 246 is the current width on v2 (I don't understand why it does not require maxWidth in v2, technically it should be full width as in v3) 🤷 

WIthout maxWidth
![image](https://github.com/user-attachments/assets/332724b2-66e4-46da-88af-71ef9e27debc)

Open for more suggestion here.

Reply:
One idea is to do it the other way around and set a reasonable minimum width to the selector root and use 'fit-content' with display 'inline-flex' for both mobile and desktop.


```
localeSelectorWrapper: {
    display: 'inline-flex',
    width: 'auto',
    maxWidth: '100%',
    marginTop: 8,
},
localeSelectorRoot: {
    width: 'fit-content',
    minWidth: '11.25rem',
    maxWidth: '100%'
},
```

Reply:
thanks for the suggestion, I tried display: 'inline-block' again, and it seems to work. I suspected it was not working because I was using it on a wrong component. This is great because we don't have to exciplit set min/max width anymore as by default the inline-block elements size themselves based on their content, similar to inline elements, but they also maintain block-like properties.